domain/company/index.js

```diff
@@ -0,0 +1,37 @@
+const buildPagination = require('../../utils/helpers/searchSpec')
+const database = require('../../database')
+const CompanyModel = database.model('company')
+const companySchema = require('../../utils/helpers/Schemas/company')
+
+const buildSearchAndPagination = buildPagination('company')
+
+class CompanyDomain {
+  async create(bodyData, options = {}) {
+    const { transaction = null } = options
+    return await CompanyModel.create(bodyData, { transaction })
+  }
+
+  async update(id, bodyData, options = {}) {
+    const { transaction = null } = options
+
+    await companySchema.validate(bodyData, { abortEarly: false })
+
+    const searchCompany = await CompanyModel.findByPk(id)
+
+    return await searchCompany.update(bodyData, { transaction })
+  }
+
+  async getById(id) {
+    return await CompanyModel.findOne({ where: { id } })
+  }
```

Comment:
nesse caso vc poderia usar o findByPk 
```suggestion
  async getById(id) {
    return await CompanyModel.findByPk(id)
  }
```